cmd/krel/cmd/release_notes.go

```diff
@@ -676,7 +676,6 @@ func releaseNotesJSON(tag string) (string, error) {
 	notesOptions.StartRev = startTag
 	notesOptions.EndRev = tag
 	notesOptions.Debug = logrus.StandardLogger().Level >= logrus.DebugLevel
-	notesOptions.ReleaseVersion = util.TrimTagPrefix(tag)
```

Comment:
This `TrimTagPrefix()` was what release-notes lacked:
https://github.com/kubernetes-sigs/release-notes/pull/188

Good riddance :)